### ISSUE TITLE
Parameter for outfile --outfile

### ISSUE THREAD

**Issue by diego-mathis** [MrKrzYch00/zopfli#17]
It would be nice if there was a param to define the output file.
While its possible to use "-c" , it seems I then do not get any info from the program.

**Comment by chrmatique** [comment]
So after lookin into this, it appears as though the help does define a `--c` flag which outputs to stdout (as defined in zopfli_bin.c:695) but doesn't allow a file name.
I'm working on a quick fork fix for this that will simply add a controlled output option :)

**Comment by MrKrzYch00** [comment]
Yes, --c outputs to stdout as per original work. The custom output filename is not supported as a command line parameter, again as per original work. Of course supporting such parameter is just a matter of some if and default value. For now, however, you could use output assigning from directory name with --dir switch. It compresses files inside the directory you pointed it to without the root directory itself:
- location: directory2020\file2020.txt
- command: zopfli --zip --dir directory2020
- result: directory2020.zip with file2020.txt inside
